lib/Reader/readerir.cpp

```diff
@@ -5863,7 +5883,11 @@ void GenIR::AddPHIOperand(PHINode *PHI, Value *NewOperand,
     LLVMBuilder->restoreIP(SavedInsertPoint);
   }
 
-  PHI->addIncoming(NewOperand, NewBlock);
+  int BlockIndex = PHI->getBasicBlockIndex(NewBlock);
+  if (BlockIndex >= 0)
+    PHI->setIncomingValue(BlockIndex, NewOperand);
+  else
+    PHI->addIncoming(NewOperand, NewBlock);
```

Comment:
It won't fail currently, but if AddPHIOperand is ever called by something else in the future, it could potentially break. I think it's safer to include it now.